src/pages/en/core-concepts/routing.md

```diff
@@ -94,6 +94,14 @@ In this example, a request for `/withastro/astro/tree/main/docs/public/favicon.s
 }
 ```
 
+## Excluding pages
```

Comment:
I prefer keeping this as "Excluding pages" because it's the goal: stop things from being built. It's also about excluding files that are components etc., not just about making pages private.

Reply:
I agree with Sarah on this one